wdfn-server/waterdata/utils.py

```diff
@@ -92,3 +92,26 @@ def wrap(f):
         return update_wrapper(func, f)
 
     return wrap
+
+
+def execute_lookup_request(url_root_cooperator_lookup, url_path_cooperator_lookup, params):
```

Comment:
This function is specific to the cooperator lookup service so consider renaming execute_lookup_request to something like execute_cooperator_lookup_request.

Reply:
All good advice. Much Appreciated.